### PR TITLE
Add floorist job/configmap to deployment

### PR DESCRIPTION
**Pull request by fijshion**
# Overview

This is a PR to add the Floorist cron job to the deployment.

## PR Checklist

- [ ] Keep PR title short, ideally under 72 characters
- [ ] Descriptive comments provided in complex code blocks
- [ ] Tests: validate optimal/expected output
- [ ] Tests: validate exceptions and failure scenarios
- [ ] Tests: edge cases
- [ ] Recovers or fails gracefully during potential resource outages (e.g. DB, Kafka)
- [ ] Documentation, if this PR changes the way other services interact with host inventory
- [ ] Links to related PRs

## Secure Coding Practices Documentation Reference

You can find documentation on this checklist [here](https://github.com/RedHatInsights/secure-coding-checklist).

## Secure Coding Checklist

- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

File: deployment.yaml [hunk 1]
```diff
@@ -549,6 +549,56 @@ objects:
...
+        volumeMounts:
+        - name: floorplan-volume
+          mountPath: "/tmp/floorplan"
+        volumes:
+          - name: floorplan-volume
+            configMap:
+              name: floorplan
+      resources:
+          limits:
+            cpu: "${CPU_LIMIT_FLOO}"
```
Comment (thearifismail) [inline deployment.yaml:597]: Are the CPU and memory limits so far off that separate variables are needed?

**Review by thearifismail — COMMENTED** [review]
Here are some questions? 

1. Why is it needed?  
2. What problem is being solved?
3. Does floorist save SQL query results or the query?  Floorist's doc says SQL queries are saved but is not clear why save queries?

File: deployment.yaml (hunk 1 above)
Comment (fijshion) [reply deployment.yaml:597]: probably not, this was copy/pasted from the compliance implementation here: https://github.com/RedHatInsights/compliance-backend/blob/32168e9d2461fdaac90bedc5e7e8bcb59c6dcc96/clowdapp.yaml#L386

Comment (kruai) [reply deployment.yaml:597]: It certainly looks like it needs fewer resources than our other deployments

**Review by kruai — APPROVED** [review]
This introduces a lot of new params, but that isn't _really_ an issue... hopefully we'll streamline it a bit later if we're able

**Review by thearifismail — APPROVED** [review]
LGTM

**Review by ryandillinfelton — APPROVED** [review]

**Comment by fijshion** [comment]
related app-interface MR: https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/27954

**Pull request merged**